cmd/commands/isbsvc_buffer_validate.go

```diff
@@ -74,7 +74,7 @@ func NewISBSvcBufferValidateCommand() *cobra.Command {
 			}
 			_ = wait.ExponentialBackoffWithContext(ctx, sharedutil.DefaultRetryBackoff, func() (bool, error) {
 				if err = isbsClient.ValidateBuffers(ctx, bfs); err != nil {
-					logger.Errorw("Buffers validation failed, will retry if the limit is not reached", zap.Error(err))
+					logger.Errorw("Buffers not created yet, will retry if the limit is not reached", zap.Error(err))
```

Comment:
1. Maybe using `Info` is better?
2. `Buffers might have not been created yet, xxxx`